packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -143,6 +166,26 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
 
     const setSubmitForm = submitFormFn => (submitForm.current = submitFormFn);
 
+    React.useEffect(() => {
+        const disposer = reaction(
+            () => buy_sell_store.form_error_code,
+            () => {
+                if (buy_sell_store.form_error_code === api_error_codes.MARKET_RATE_CHANGE) {
+                    if (!isMobile()) {
```

Comment:
can we do if(isMobile()) first so it's easier to read?

Reply:
I can use `isDesktop` alsoo